inspect/inspect.go

```diff
@@ -0,0 +1,174 @@
+package inspect
```

Comment:
could we get a doc.go for this package as well?

Reply:
Done! PTAL